### PR TITLE
feat(gatsby-source-contentful): improve gatsby-plugin-image errors

### PR DESCRIPTION
**Pull request by wardpeet**
## Description

Show proper error when gatsby-plugin-image is not available or configured.

With this PR:
image

image

Before:
image

### CODE REVIEW

File: packages/gatsby-source-contentful/src/source-nodes.js [hunk 1]
```diff
@@ -80,9 +80,12 @@ export async function sourceNodes(
     `environment`
   )}`
 
-  const fetchActivity = reporter.activityTimer(`Contentful: Fetch data`, {
-    parentSpan,
-  })
+  const fetchActivity = reporter.activityTimer(
```
Comment (wardpeet) [inline packages/gatsby-source-contentful/src/source-nodes.js:83]: These edits got removed by rebasing in image-cdn PR

File: packages/gatsby-source-contentful/src/create-schema-customization.js [hunk 2]
```diff
@@ -96,9 +96,39 @@ export async function createSchemaCustomization(
...
+  let getGatsbyImageFieldConfig = null
+  try {
+    const graphqlUtils = await import(`gatsby-plugin-image/graphql-utils`)
+    getGatsbyImageFieldConfig = graphqlUtils.getGatsbyImageFieldConfig
+  } catch (err) {
+    const isYarn = process.env.npm_config_user_agent.includes(`yarn/`)
+
+    let command = `npm install gatsby-plugin-image`
+    if (isYarn) {
+      command = `yarn add gatsby-plugin-image`
```
Comment (LekoArts) [inline packages/gatsby-source-contentful/src/create-schema-customization.js:108]: I don't think other error messages care about this so I'm leaning a bit towards removing this as our docs stance is also do only ever show npm (as with that you'd then also need to support pnpm etc.)

File: packages/gatsby-source-contentful/src/create-schema-customization.js [hunk 3]
```diff
@@ -96,9 +96,39 @@ export async function createSchemaCustomization(
...
+    }
+
+    reporter.panic({
+      id: CODES.GatsbyImageUtils,
+      context: {
+        sourceMessage: `Gatsby-plugin-image is missing from your project.\n\nPlease install it with: ${command}.`,
```
Comment (LekoArts) [inline packages/gatsby-source-contentful/src/create-schema-customization.js:114]:
```suggestion
        sourceMessage: `gatsby-plugin-image is missing from your project.\n\nPlease install it with: ${command}.`,
```

File: packages/gatsby-source-contentful/src/create-schema-customization.js [hunk 4]
```diff
@@ -96,9 +96,39 @@ export async function createSchemaCustomization(
...
+  // if gatsby-plugin-image is not installed
+  if (
+    !store
+      .getState()
+      .flattenedPlugins.find(plugin => plugin.name === `gatsby-plugin-image`)
+  ) {
+    reporter.panic({
+      id: CODES.GatsbyImageUtils,
+      context: {
+        sourceMessage: `Gatsby-plugin-image is missing from your gatsby-config file.\n\nPlease add "gatsby-plugin-image" to your plugins array.`,
```
Comment (LekoArts) [inline packages/gatsby-source-contentful/src/create-schema-customization.js:128]:
```suggestion
        sourceMessage: `gatsby-plugin-image is missing from your gatsby-config file.\n\nPlease add "gatsby-plugin-image" to your plugins array.`,
```

**Review by LekoArts — CHANGES_REQUESTED** [review]
Two smaller nits

**Review by axe312ger — APPROVED** [review]
LGTM! Thanks!

**Review by LekoArts — APPROVED** [review]

**Pull request merged**